test/conftest.py

```diff
@@ -152,3 +152,26 @@ def ZoneInfo(zoneinfo):
 def utc(request):
     """Return all the UTC implementations."""
     return request.param
+
+class DoctestZoneInfo(_zoneinfo.ZoneInfo):
+    """Constent ZoneInfo representation for tests."""
+    def __repr__(self):
+        return f"ZoneInfo(key={self.key!r})"
+
+def doctest_print(obj):
+    """doctest print"""
+    if isinstance(obj, bytes):
+        obj = obj.decode("UTF-8")
+    print(str(obj).strip().replace("\r\n", "\n").replace("\r", "\n"))
+
+@pytest.fixture()
+def env_for_doctest(monkeypatch):
+    """Modify the environment to make doctests run."""
+    monkeypatch.setitem(sys.modules, "zoneinfo", _zoneinfo)
+    monkeypatch.setattr(_zoneinfo, "ZoneInfo", DoctestZoneInfo)
+    from icalendar.timezone.zoneinfo import ZONEINFO
```

Comment:
This fails using icalendar v5.

Reply:
Maybe, we can create a test to check version 5, too.
Icalendar v5 and v6 are not so much different when it comes to implementation.